project/src/main/puzzle/line-clearer.gd

```diff
@@ -108,13 +108,13 @@ func _physics_process(_delta: float) -> void:
 		var old_lines_being_erased := lines_being_erased
 		var old_lines_being_deleted := lines_being_deleted
 		
-		set_physics_process(false)
-		
 		lines_being_cleared = []
 		lines_being_erased = []
 		lines_being_deleted = []
 		
 		_delete_lines(old_lines_being_cleared, old_lines_being_erased, old_lines_being_deleted)
+		
+		set_physics_process(false)
```

Comment:
'is_clearing_lines' will return true during this entire process, but 'is_box_building' will return false during the final signal. For consistency, we should think more carefully about when we call 'set_physics_process(false)' and add a comment.

Reply:
Added comment 'causes 'Playfield.is_building_boxes()' to return false...' and modified BoxBuilder for consistency.